### PR TITLE
Routine updates for JDK11 build compatibility

### PR DESCRIPTION
**Pull request by Vlatombe**
* Update parent POM
* Fix a spotbugs issue

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +24,8 @@
     </developers>
 
     <properties>
-      <jenkins.version>1.609.1</jenkins.version>
+        <jenkins.version>2.204.6</jenkins.version>
```
Comment (bitwiseman) [inline pom.xml:27]:
```suggestion
        <jenkins.version>2.249.3</jenkins.version>
```

**Review by bitwiseman — APPROVED** [review]

**Comment by batmat** [comment]
Build failing, retriggering to see if this is transient.

**Comment by mikecirioli** [comment]
I am able to reproduce these failures locally using windows, investigating..

**Comment by mikecirioli** [comment]
superseded by #37 i believe

**Pull request closed without merging**